src/components/Burger/burger.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+import {useState}  from 'react';
+import "./burger.css";
+
+const Burger = () => {
```

Comment:
When naming components try to be as descriptive as possible, what is a Burger component?
MobileNav could be a better name or something like that, that way other developers who work with you/get the codebase after you can understand what this component does without opening it.

Reply:
Already changed to BurgerMenu :)
Thanks!